drivers/netdev_ieee802154_submac/Makefile.include

```diff
@@ -0,0 +1 @@
+PSEUDOMODULES += netdev_ieee802154_submac_auto_ack
```

Comment:
```suggestion
PSEUDOMODULES += netdev_ieee802154_submac_soft_ack
```

Auto-ACK will work on hard-MAC devices regardless of the module, this is about sending ACKs in software